mage_ai/frontend/components/ServerTimeDropdown/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { 
+  LOCAL_STORAGE_KEY_DISPLAY_LOCAL_SERVER_TIME, LOCAL_STORAGE_KEY_INCLUDE_SERVER_TIME_SECONDS, 
+} from '@storage/constants';
+import { get, set } from '@storage/localStorage';
+
+function getLocalStorageValue(storageKey: string): boolean {
+  const storageValue = get(storageKey, null);
```

Comment:
If you just called `get(storageKey, false)` which returns a default `false` value, would this `getLocalStorageValue` method still be needed?

Reply:
ah true, that's a good simplification